src/psyclone/tests/psyir/frontend/fparser2_do_handler_test.py

```diff
@@ -89,44 +90,34 @@ def test_do_construct(parser):
     assert isinstance(new_loop.loop_body[0], Assignment)
 
 
-@pytest.mark.usefixtures("f2008_parser")
-def test_do_construct_while():
-    ''' Check that do while constructs are placed in Codeblocks. '''
+def test_do_construct_while(parser):
```

Comment:
Now that I see this test, I take back what I said on Teams (sorry). I think it's fine to use the `fortran_reader` fixture here and just get the PSyIR from that, rather than explicitly using the parser.

Reply:
Done.